src/examples/example_client.cpp

```diff
@@ -63,23 +61,25 @@ int main(int argc, char** argv)
     for (std::string d : ns.As<std::vector<std::string>>())
       std::cout << "    "  << d << std::endl;
     
-    //Get namespace index we are interrested in
-    uint32_t idx = client.GetNamespaceIndex("http://examples.freeopcua.github.io");
+    //Get namespace index we are interested in
+    // uint32_t idx = client.GetNamespaceIndex("http://examples.freeopcua.github.io");
     
-    //Get Node using path (BrowsePathToNodeId call)
-    std::vector<std::string> varpath({std::to_string(idx) + ":NewObject", "MyVariable"});
-    //std::vector<std::string> varpath({"Objects", "5:Simulation", "5:Random1"}); //Example data from prosys server
-    OpcUa::Node myvar = objects.GetChild(varpath);
-    std::cout << "got node: " << myvar << std::endl;
+	OpcUa::Node dynamicNode = client.GetNode(OpcUa::NodeId("Demo.Dynamic.Scalar.Float", 2));
```

Comment:
this is is specific to your server and should not be commited. Please review all your changes in example_client.cpp. Some are good but some are specific to your project,